kernel/dev/vga/graphics.hpp

```diff
@@ -50,4 +53,17 @@ void pixel(uint32_t x, uint32_t y, uint32_t color);
  */
 void putrect(uint32_t x, uint32_t y, uint32_t w, uint32_t h, uint32_t color);
 
+/**
+ * @brief Fill the backbuffer with '0'.
+ *
+ */
+void clearbuffer();
+
+/**
+ * @brief Swap the data on backbuffer to memory video buffer
+ * and show in the screen.
+ *
+ */
+void show();
```

Comment:
Personally I would call this `update` instead of `show`.

Reply:
`swap` is the "official" term

Reply:
good suggestions guys,

wouldn't 'swap' be bad for the "end user programmer" later?

`update` or `updatebuffer` sounds good for me.

Reply:
There really won't be an "end user" for the kernel. Eventually all user programs will make system calls, so the only time this will ever get called is by kernel code. Make sense?

Reply:
yeap totally 
so i change it to `swap`?
`bf::swap()`

Reply:
I let it 'swap'